### DISCUSSION TITLE
A correct way to open Ark UI DatePicker on input focus

### DISCUSSION THREAD

**Discussion by markoffden** [chakra-ui/ark#1821] [Q&A]
Hi, I was wondering what is the best practice to implement "open on input focus" DatePicker behaviour. Thanks

**Comment by amegalulb** [comment]
hi, did you manage to resolve the issue with opening to input focus?

**Reply by amegalulb** [reply]
If I understand correctly, if the input is in focus, then the next time you click on the menu it will simply close it and it is impossible to select a date or move further by month. Have you encountered the same behavior?

**Reply by amegalulb** [reply]
For example, if you go to https://ark-ui.com/react/docs/components/date-picker and
1. click on the trigger button
2. focus input
3. click somewhere on the menu

The menu just closes. @segunadebayo sorry for tag, but can you or someone else explain is this correct behaviour?

**Reply by cschroeter** [reply]
I could reproduce the issue. Lets see how we can fix this

**Reply by amegalulb** [reply]
Thank you!! Do u i need to create an issue for this?

**Reply by s-titov** [reply]
any news here?
